### ISSUE TITLE
[Bug] Settings fails to save

### ISSUE THREAD

**Issue by necrophili** [MeteorDevelopment/meteor-client#1706]
### Describe the bug

asfdhjklm da settings for sum modules (not only color settings) are failing to save 

### Steps to reproduce

```bash
try to save some color settings or some other settings, reload the profile asldfjasdf settings = gone. bro :sob: :cryingrn:
```

### Meteor Version

1339

### Minecraft Version

1.17.1

### Operating System

macOS

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by necrophili** [comment]
fyi this issue isnt recreateable everytime, sometime you have to load the profiles multiple time or relog

**Comment by RedCarlos26** [comment]
Happens to me as well, for logout spots especially based on my experience

**Comment by MineGame159** [comment]
Fixed in 4316e128240021d35058dde7f0d6ad2b0b8c14b5.
